<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>391</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>111</number>
    </property>
    <property name="horizontalSpacing">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>30</number>
    </property>
    <item row="0" column="4">
     <widget class="QPushButton" name="search_button">
      <property name="text">
       <string>Найти</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QRadioButton" name="anonymous_rb">
      <property name="text">
       <string>создать случайно</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="search_field"/>
    </item>
    <item row="0" column="2">
     <widget class="QRadioButton" name="non_anonymous_rb">
      <property name="text">
       <string>найти профиль</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QComboBox" name="results_cbox"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="start_chat_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать чат!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nick_field">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nick_label">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>мой ник:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="use_default_nick_chbox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>исп. стандартный</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
